scripts/check_compliance.py

```diff
@@ -156,6 +156,9 @@ def run(self):
         os.environ["BOARD_DIR"] = "boards/*/*"
         os.environ["ARCH"] = "*"
         os.environ["PROJECT_BINARY_DIR"] = tempfile.gettempdir()
+        os.environ["PYTHON_EXECUTABLE"] = os.path.normpath(sys.executable)
```

Comment:
This is actually not needed by anything but is set to make the test which checks which env variables are needed to be set by `kconfig.cmake` so that when external/third-party scripts want to call cmake they can use the same lookup list as this test is using `kconfig_env_vars.txt`at least that's the idea behind of the change.